packages/page-accounts/src/Accounts/index.tsx

```diff
@@ -60,6 +60,7 @@ function Overview ({ className = '', onStatusChange }: Props): React.ReactElemen
   const [filterOn, setFilter] = useState<string>('');
   const [sortedAccountsWithDelegation, setSortedAccountsWithDelegation] = useState<SortedAccount[] | undefined>();
   const [{ sortedAccounts, sortedAddresses }, setSorted] = useState<Sorted>({ sortedAccounts: [], sortedAddresses: [] });
+  const [{ sortBy, sortFromMax }, setSortBy] = useState<{ sortBy: SortCategory, sortFromMax: boolean }>({ sortBy: 'date', sortFromMax: true });
```

Comment:
could you extract some const like DEFAULT_ACCOUNTS_SORT or similar?

Reply:
Sure will do